ironfish-rust-nodejs/index.js

```diff
@@ -275,6 +275,10 @@ module.exports.DECRYPTED_NOTE_LENGTH = DECRYPTED_NOTE_LENGTH
 module.exports.Note = Note
 module.exports.TransactionPosted = TransactionPosted
 module.exports.PROOF_LENGTH = PROOF_LENGTH
+module.exports.TRANSACTION_SIGNATURE_LENGTH = TRANSACTION_SIGNATURE_LENGTH
+module.exports.TRANSACTION_PUBLIC_KEY_LENGTH = TRANSACTION_PUBLIC_KEY_LENGTH
```

Comment:
Why is the transaction public key different than the account public key length?

Reply:
Agreed. The [comment](https://github.com/iron-fish/ironfish/pull/2890/files#diff-347e78ce8439f128b344eb339befd4f4899d395e0cc65c953d0fb1426aa237f0R57) also indicates there's an existing constant that can be used.

Reply:
This is randomized public key of the sender for a transaction. For the account public key, are you referring to public address? if so that is different thing.

Reply:
Ah yeah, my bad forgot about that. What about using `RANDOMIZED_PUBLIC_KEY_LENGTH` then?

Reply:
I will rename to `TRANSACTION_PUBLIC_KEY__RANDOMNESS_LENGTH`